config/links_out.json

```diff
@@ -0,0 +1,167 @@
+{
```

Comment:
We could maybe represent this configuration in YAML instead of JSON?

Doing so would allow us to use Rails' [built in mechanism to load configuration files](https://guides.rubyonrails.org/configuring.html#custom-configuration).

```rb
module MyApp
  class Application < Rails::Application
    config.taxonomy_navigation_links_out = config_for(:taxonomy_navigation_links_out)
  end
end
```

```yaml
---
default: &default
  content_purpose_supergroup:
    news_and_communications:
    - title: policy_and_engagement
      type: content_purpose_supergroup
      supergroup: policy_and_engagement
    - title: research_and_data
      type: content_purpose_subgroup
      supergroup: transparency
    - title: guidance
      type: content_purpose_subgroup
      supergroup: guidance_and_regulation
    - title: news
      type: content_purpose_subgroup
      supergroup: news_and_communications
    - title: incidents
      type: content_purpose_subgroup
      supergroup: transparency
    guidance_and_regulation:
    - title: services
      type: content_purpose_supergroup
      supergroup: services
    - title: guidance
      type: content_purpose_subgroup
      supergroup: guidance_and_regulation
    - title: news
      type: content_purpose_subgroup
      supergroup: news_and_communications
    - title: decisions
      type: content_purpose_subgroup
      supergroup: transparency
    - title: step_by_step
      type: document_type
      supergroup: services
    services:
    - title: services
      type: content_purpose_supergroup
      supergroup: services
    - title: guidance
      type: content_purpose_subgroup
      supergroup: guidance_and_regulation
  content_purpose_subgroup:
    policy:
    - title: policy_and_engagement
      type: content_purpose_supergroup
      supergroup: policy_and_engagement
    - title: policy
      type: content_purpose_subgroup
      supergroup: policy_and_engagement
    incidents:
    - title: incidents
      type: content_purpose_subgroup
      supergroup: transparency
    - title: news
      type: content_purpose_subgroup
      supergroup: news_and_communications
    - title: research_and_data
      type: content_purpose_supergroup
      supergroup: transparency
    - title: guidance
      type: content_purpose_subgroup
      supergroup: guidance_and_regulation
    research_and_data:
    - title: research_and_data
      type: content_purpose_supergroup
      supergroup: transparency
    decisions:
    - title: news
      type: content_purpose_subgroup
      supergroup: news_and_communications
    - title: guidance
      type: content_purpose_subgroup
      supergroup: guidance_and_regulation
  document_type:
    fatality_notice: []
    manual: []
    manual_section: []
    open_consultation:
    - title: policy_and_engagement
      type: content_purpose_supergroup
      supergroup: policy_and_engagement
    - title: decisions
      type: content_purpose_subgroup
      supergroup: transparency
    closed_consultation:
    - title: policy_and_engagement
      type: content_purpose_supergroup
      supergroup: policy_and_engagement
    - title: decisions
      type: content_purpose_subgroup
      supergroup: transparency
    consultation_outcome:
    - title: decisions
      type: content_purpose_subgroup
      supergroup: transparency

development:
  <<: *default

test:
  <<: *default

production:
  <<: *default
```